holidays/countries/french_polynesia.py

```
@@ -0,0 +1,48 @@
+#  holidays
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)